### DISCUSSION TITLE
This is a question regarding the development of betting_robot using selenium.

### DISCUSSION THREAD

**Discussion by 0xIntelTech** [community/community#135402] [Programming Help] [answered]
### Select Topic Area

Question

### Body

I would like to send the user's betting information to the desired email and phone number in the form of a text message within 5 minutes. I would like to receive a solution regarding development.

**Comment by 0xpuck** [comment, marked as answer]
You can send emails using Simple Mail Transfer Protocol.
SMTP(Simple Mail Transfer Protocol) is a communication protocol used for sending and receiving email messages over the Internet. Mail servers and other message transfer agents (MTAs) use SMTP to send, receive, and relay mail messages.

Below is a simple example implemented in python.
=================================
```
import smtplib
from email.mime.multipart import MIMEMultipart
from email.mime.text import MIMEText

email_username = 'email_username'
email_passkey = 'email_passkey'

# Create the email
msg = MIMEMultipart()
msg['To'] = 'email'
msg['Subject'] = 'Betting Support'

msg['From'] = f'"888champs.com Betting Info" <{email_username}>'
body = '<p>Input the text for the mail</p>'

msg.attach(MIMEText(body, 'html'))

try:
    # Set up the SMTP server
    server = smtplib.SMTP('smtp.gmail.com', 587)
    server.starttls()
    server.login(email_username, email_passkey)
    
    # Send the email
    server.send_message(msg)
except Exception as error:
    raise Exception('Error sending Currently BettingInfo email')
finally:
    server.quit()
```
=================================
I hope it helps.

**Reply by 0xIntelTech** [reply]
Thank you for your answer.
Could you explain how to get "email_passkey"?

**Reply by 0xIntelTech** [reply]
I have successfully sent a text message to an email.
The code you provided helped me a lot.
And I successfully managed to send an email to a phone number on my own.
Thanks again for your passion.

**Comment by 0xIntelTech** [comment]
If anyone knows how to get email_passkey when using SMTP (Simple Mail Transfer Protocol), please help

**Reply by 0xpuck** [reply]
I will prepare a reply and send it to you as soon as possible.

**Comment by 0xpuck** [comment]
**How to create a new passkey for Google**
What you'll need: To make this work, you'll only need a valid Google account and a web browser (I would recommend using Chrome). Do note that the device you use must be either a mobile device or a laptop/desktop with biometrics (such as a fingerprint scanner).

**1. Log into your Google account**
The first thing to do is to open your web browser and log into your Google account. If you've already done that, you're good to go; otherwise, you can open Gmail, Drive, Google Calendar (or any of the Google services) and log in.

**2. Open the Passkeys manager**
Next, head over to [myaccount.google.com/signinoptions/passkeys](https://myaccount.google.com/signinoptions/passkeys). When the page loads, you'll be prompted for your account password once again.

**3. Create a new Passkey**
On the resulting page, click Create a Passkey. You'll, once again, be prompted for your Google account password.
![Screenshot 2024-08-11 045115](https://github.com/user-attachments/assets/952cae34-5720-41ab-8be3-d582be8f6541)

_If you've already created passkeys on your mobile devices, they'll be listed here._

**4. Continue with the Passkey creation**
In the resulting pop-up, click Continue.
![Screenshot 2024-08-11 045312](https://github.com/user-attachments/assets/e4855de3-7fd8-423e-ba48-8bb06028882b)

_You are about to create a passkey for your Google account._

You will be prompted to allow Chrome to access and use your passkeys. When prompted, click Allow. If you're using MacOS, you'll need to make sure to turn on iCloud Keychain (if it's not already enabled). Once you've done that, you'll be prompted for your fingerprint and then the passkey is done and ready to use.

**Reply by 0xIntelTech** [reply]
thank you bro
